addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -68,7 +67,6 @@ var MockMailService = Class.extend({
         return MailService;
     },
     messaging() {
```

Comment:
still needed if service is returned as it is?

Reply:
yes, tests that make use of legacy helper Mail.getServices require messaging service instead, so it should stay there.